modules/pipeline/providers/cron/compensator/provider.go

```diff
@@ -186,7 +230,7 @@ func (p *provider) cronInterruptCompensate(pc spec.PipelineCron) error {
 
 	if len(needTriggerTimes) > 0 {
 		// Pipeline search record created based on createlyname + source
-		existPipelines, _, _, _, err := p.client.PageListPipelines(apistructs.PipelinePageListRequest{
```

Comment:
better to define a struct, too many returns

Reply:
adjusted in another pr.